soot-infoflow/src/soot/jimple/infoflow/InfoflowConfiguration.java

```diff
@@ -1293,6 +1293,7 @@ public boolean equals(Object obj) {
 	private boolean oneSourceAtATime = false;
 
 	private static String baseDirectory = "";
+	private static boolean createActivityEntryMethods = true;
```

Comment:
Das sollte in der ``InfoflowAndroidConfiguration`` drin sein, weil es um Android-spezifische Settings geht.